pmg/static/resources/css/style.scss

```diff
@@ -1143,25 +1143,47 @@ body.email_alerts {
   a[href]:after {
     content: none;
   }
-  .email-alert-box, .correct-this-page-footer, .footer, .top-links, .cte-list-search, .nav, .premium, img, .cte-about-link, .cte-dtl-meetings-search, .cte-dtl-meetings-filter, .space-bottom, .bill-status-container, .bill-versions, #wrapper > div > article > div.row > div.col-sm-4 > div, .form-group, #wrapper > div > article > div.row > div.col-sm-8 > div > div > div > div.media-body > div.row, #wrapper > div > article > div.row > div.col-sm-8 > header > p, .pull-left {
+
+  .email-alert-box,
+  .correct-this-page-footer,
+  .footer,
+  .cte-list-search,
+  nav,
+  .nav,
+  .premium,
+  .breadcrumb,
+  .cte-about-link,
+  .cte-dtl-meetings-search,
+  .cte-dtl-meetings-filter,
+  .space-bottom,
```

Comment:
I'm a little concerned about this one, because it's used to make vertical spacing work correctly in some cases. I don't know why we'd want to remove that when printing?